build.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+IMAGE_NAME="meta-aos-rpi"
+CONTAINER_NAME="meta-aos-rpi_container"
+ARTIFACTS_DIR="$(pwd)/artifacts"
+
+# List aos-rpi.yaml parameters.
+DOMD_NODE_TYPE="main"
+MACHINE="rpi5"
+DOMD_ROOT="usb"
+SELINUX="disabled"
+
+error_and_exit() {
+    echo "$1"
```

Comment:
1. Should be redirected to stderr.
2. Should be echo "Error: $1" And called without "Error: ..." respectively.